tests/test_pyrate.py

```diff
@@ -323,12 +328,12 @@ def test_stack_rate_equal(self):
 
 
 @pytest.mark.slow
-@pytest.mark.skipif(not PYTHON3P8, reason="Only run in python 3.8")
+@pytest.mark.skipif(not PY37GDAL304, reason="Only run in python 3.8")
```

Comment:
The skip reason doesn't match what's being done anymore... is this intended?

Reply:
hey, thank you for the review. Yes, it's an oversight from me. We just redistributed the long running tests into the various environements, but I will change the text!